app/forms/publish/course_rollover_form.rb

```diff
@@ -0,0 +1,25 @@
+module Publish
+  class CourseRolloverForm
```

Comment:
Can we have a test for this class please

Reply:
Done